R/tm_a_regression.R

```diff
@@ -1026,7 +1033,41 @@ srv_a_regression <- function(id,
       )
     })
 
-    # Render R code.
+    # # Render R code.
+    # subset_code <- function(code, data) {
+    #   gsub(code, "", teal.data::get_code(data), fixed = TRUE)
+    # }
+
+    setup_code_r <- reactive(teal.data::get_code(req(data())))
+    libraries_code_r <- reactive(teal.code::get_code(req(decorated_output_q()), labels = "libraries"))
```

Comment:
Makes sense. Let's not focus on the code-splitting and finish the fundamental part of teal.reporter. We can come back to this discussion at some point